notebooks/source/circulant_gp.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
<p>It would be nice to put the legends outside the plots as in <a href="https://stackoverflow.com/questions/4700614/how-to-put-the-legend-outside-the-plot" target="_blank">https://stackoverflow.com/questions/4700614/how-to-put-the-legend-outside-the-plot</a> :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>